tasks.py

```diff
@@ -0,0 +1,206 @@
+            --conda-env-dir {conda_prefix}
+    """)
+
+
+def _get_test_command():
+    if sys.platform.startswith('win'):
+        test_command = strip_and_join(f"""
+            {BUILD_DIR_DEFAULT}\\test\\tests
+                --success
+                --reporter compact
+        """)
+    else:
+        test_command = strip_and_join(f"""
+            {BUILD_DIR_DEFAULT}/test/tests
+                --success
+                --reporter compact
+        """)
+    return test_command
+
+
+if sys.platform.startswith('win'):
+    @task
+    def msvc(c, clean=False, config='Release'):
+        """
+        Generates a Visual Studio project at the "build/msvc" directory.
+        Assumes that the environment is already configured using:
+            conda devenv
+            activate autodiff
+        """
+        build_dir, artifacts_dir = _get_and_prepare_build(
+            c,
+            clean=clean,
+            build_subdirectory=BUILD_DIR_DEFAULT / "msvc",
+        )
+        cmake_command = _get_cmake_command(build_dir=build_dir, cmake_generator="Visual Studio 15 2017",
+                                           cmake_arch="x64", config=config)
+        os.chdir(build_dir)
+        c.run(cmake_command)
+
+
+@task
+def compile(c, clean=False, config='Release', number_of_jobs=-1, verbose=False, gen_wrappers=False):
+    """
+    Compiles autodiff by running CMake and building with `ninja`.
+    Assumes that the environment is already configured using:
+        conda devenv
+        [source] activate autodiff
+    """
+    build_dir = _get_and_prepare_build(
+        c,
+        clean=clean,
+        build_subdirectory=BUILD_DIR_DEFAULT,
+    )
+
+    cmake_command = _get_cmake_command(build_dir=build_dir, cmake_generator="Ninja", config=config, verbose=verbose)
+    build_command = strip_and_join(f"""
+        cmake
+            --build .
+            --target install
+            --config {config}
+            --
+                {f"-j {number_of_jobs}" if number_of_jobs >= 0 else ""}
+                {"-d keeprsp" if sys.platform.startswith("win") else ""}
+    """)
+
+    commands = [cmake_command, build_command]
+    if gen_wrappers:
+        wrappers_command = _get_wrappers_command(build_dir / "wrappers/conda")
+        commands.append(wrappers_command)
+
+    if sys.platform.startswith('win'):
+        for vcvars_path in _get_vcvars_paths():
+            if not vcvars_path.is_file():
+                continue
+            commands.insert(0, f'"{vcvars_path}" amd64')
+            break
+        else:
+            raise Exit(
+                'Error: Commands to configure MSVC environment variables not found.',
+                code=1,
+            )
+
+    os.chdir(build_dir)
+    c.run("&&".join(commands))
+
+
+@task
+def clear(c, build_dir_path=BUILD_DIR_DEFAULT):
+    """
+    Clear autodiff build directory
+    """
+    remove_directory(build_dir_path)
+
+
+@task
+def wrappers(c, wrappers_dir=BUILD_DIR_DEFAULT / "wrappers/conda"):
```

Comment:
The `wrappers` task is useful because it creates symlinks to conda binaries. So instead of search for binaries in the `miniconda` installation dir in your file system, this task wraps everything for you to a given directory. This directory is `(BUILD_DIR)/wrappers/conda` by default. It is useful, at least for me, because helps me to configure my IDE (CLion, pointing which cmake to use, which python, compilers and etc. The same is valid for VS Code).

Usage:
```bash
inv wrappers (options)
```

Options:
* `-w` or `--wrappers-dir`. Specify the dir you would like to create the wrappers (symlinks). Default is the default build dir